docs-pipeline.md

```diff
@@ -0,0 +1,169 @@
+# How content changes in enki
+
+This describes the overall steps that content goes through.
+Several of these can be clumped together instead of being separate and would reduce the amount of validation & debugging/testing.
+
+* [Overall Process](#overall-process)
+* [What happens in each step](#what-happens-in-each-step)
+   * [git-fetch](#git-fetch)
+   * [git-fetch-metadata](#git-fetch-metadata)
+      * [fetch-update-metadata](#fetch-update-metadata)
+      * [fetch-map-resources](#fetch-map-resources)
+   * [git-assemble](#git-assemble)
+   * [git-bake](#git-bake)
+   * [git-bake-meta](#git-bake-meta)
+   * [git-link](#git-link)
+   * [Output-specific](#output-specific)
+* [Required languages](#required-languages)
+   * [Validation](#validation)
+
+
+# Overall Process
+
+Listed here is the pipeline steps grouped together into steps that could be combined.
+
+1. [git-fetch](#git-fetch): Clone URL & checkout commit
+1. [git-fetch-metadata](#git-fetch-metadata), [git-assemble](#git-assemble)
+    - Replace `<md:metadata>` and move images to `../resources/{sha}`, Convert CNXML to HTML and assemble all the files together
+1. [git-bake](#git-bake)
+1. [git-bake-meta](#git-bake-meta), [git-link](#git-link)
+    - Create a book metadata JSON file with slugs and abstracts, add attributes to links for REX so it knows the canonical book
+1. Output-specific steps
+
+
+# What happens in each step
+
+## git-fetch
+
+This just runs an authenticated `git clone` and checks out the correct branch/commit.
+
+**Validation:**
+
+Use POET CLI to validate the results.
+
+
+## git-fetch-metadata
+
+3 things happen.
```

Comment:
"3" here should probably be written out to avoid parsing it as an out-of-place `ol` item